incubator/azuremonitor-containers/README.md

```diff
@@ -86,7 +106,9 @@ $ helm install --name myrelease-1 -f values.yaml incubator/azuremonitor-containe
 
 ```
 
-After you successfully deploy the chart, you will be able to see your data in the [azure portal](aka.ms/azmon-containers)
+After you successfully deploy the chart, you will be able to see your data in
+- [azure public cloud portal](https://aka.ms/azmon-containers) for the clusters in Azure Public Cloud
+- [azure china cloud portal](https://aka.ms/multi-cluster-mooncake) for the clusters in Azure China Cloud
```

Comment:
Should we include US Gove here as well ? As we are adding it in other places here ? May be just create a link and we can populate as soon as we have it available ?